istioctl/cmd/kubeinject_test.go

```diff
@@ -87,3 +95,59 @@ func TestKubeInject(t *testing.T) {
 		})
 	}
 }
+
+func TestKubeInjectWithoutWebhook(t *testing.T) {
```

Comment:
Is it possible to fold this test as an additional test case in the above test?

Reply:
Revised.